components/fade/index.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components'
+
+const FadeOut = styled.div`
+  display: inline-block;
+  animation: ${(props) => (props.show ? 'none' : 'fadeOut 0.5s linear')};
+  opacity: ${(props) => (props.show ? 1 : 0)};
```

Comment:
This hides the contents visually, but it's still in the accessibility tree (ie can still be accessed via screen reader). When it's not visible, something needs to be added here that prevents screen readers from getting to it. Same with the FadeIn component below.

Reply:
Added a display:none to the FadeOut component. The FadeIn component already has a display:none so that should be ok. In the case of the form element, it is removed from the DOM entirely once isSubmitted is true.